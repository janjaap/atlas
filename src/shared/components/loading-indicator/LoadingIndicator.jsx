import React from 'react';

import './_loading-indicator.scss';

const LoadingIndicator = () => (
  <span className="loading-indicator" />
);

export default LoadingIndicator;
