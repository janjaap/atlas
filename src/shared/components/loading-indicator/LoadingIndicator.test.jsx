import React from 'react';
import { shallow } from 'enzyme';

import LoadingIndicator from './LoadingIndicator';

describe('Checkbox', () => {
  it('should render everything', () => {
    const wrapper = shallow(
      <LoadingIndicator />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
