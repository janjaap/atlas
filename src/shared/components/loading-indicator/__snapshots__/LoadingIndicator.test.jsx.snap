// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should render everything 1`] = `
<span
  className="loading-indicator"
/>
`;
